src/shogun/classifier/LDA.h

```diff
@@ -82,6 +82,8 @@ template <class ST> class CDenseFeatures;
  *
  * <em>::AUTO_LDA</em> : This mode automagically chooses one of the above modes for
  * the users based on whether N > D (chooses ::FLD_LDA) or N < D(chooses ::SVD_LDA)
+ * Note that even if N > D FLD_LDA may fail being the covariance matrix not invertible,
```

Comment:
What happens when this is the case? How is the user informed? Error message? Or automatic fallback?